.github/workflows/release-prod.yml

```diff
@@ -80,7 +80,7 @@ jobs:
         run: pnpm install
 
       - name: Build everything
-        run: pnpm build
+        run: PRIVATE_FONTS=true FONTS_REPO_ACCESS_TOKEN=${{ secrets.FONTS_REPO_ACCESS_TOKEN }} pnpm build
```

Comment:
i've added these in github and vercel.

however, i tested this with the `staging` deployment, and i can see [this line](https://vercel.com/recallnet/comps/31Nb176EWsL9tfNMYvzzhRzgZxs5#L314) says it's using the "open" and not "private" fonts (this comes from the `init.sh` script):
```
Copied open fonts TS template to src/index.ts.
```
regardless, we can see if this CI workflow change fixes it up in prod since the deployment process is slightly different than staging.